lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -831,6 +831,50 @@ public static function get_style_variations_from_directory( $directory, $scope =
 		return $variations;
 	}
 
+	/**
+	 * Resolves relative paths in theme.json typography to theme absolute paths
+	 * and returns them in an array that can be embedded
+	 * as the value of `_link` object in REST API responses.
+	 *
+	 * @since 6.6.0
+	 *
+	 * @param 
+	 * @return array An array of resolved paths.
+	 */
+	private static function get_resolved_fonts_theme_uris( $theme_json_data ) {
```

Comment:
In don't think that the current design of that kind of function (a very long function processing all the different part of the json data) in both the backend and frontend are the best because they are hard to follow and update because they are very long. Would those function benefit of being more modular?

Reply:
Still to be in line with the rest of the codebase I moved the functionality inside `get_resolved_theme_uris` here: https://github.com/WordPress/gutenberg/pull/65019/commits/efd95620151a7dba754fb1d1d4407be34ecbdabb